Doc/library/pprint.rst

```diff
@@ -44,9 +136,9 @@ The :mod:`pprint` module defines one class:
    Construct a :class:`PrettyPrinter` instance.  This constructor understands
    several keyword parameters.
 
-   *stream* (default ``sys.stdout``) is a :term:`file-like object` to
+   *stream* (default :data:`sys.stdout`) is a :term:`file-like object` to
    which the output will be written by calling its :meth:`!write` method.
-   If both *stream* and ``sys.stdout`` are ``None``, then
+   If both *stream* and :data:`sys.stdout` are ``None``, then
```

Comment:
In order to reduce visual noise, we only produce a link for _one_ occurrence[^1] of a "referenceable item".

```suggestion
   If both *stream* and :data:`!sys.stdout` are ``None``, then
```

[^1]: normally the first one

Reply:
Noted and addressed, thanks